test/unittests/api/v0.0.1/Candidates_Test.php

```diff
@@ -97,17 +96,52 @@ function testPostCandidateValid() {
                     ]
                 ]
                 );
+            return $API;
         } catch(\Loris\API\SafeExitException $e) {
             $API = $e->Object;
+            return $API;
         }
+    }
+    function testPostCandidateValid() {
```

Comment:
The code calls creation successfully after whatever existing validation checks there are. Beyond that, it's random generation code that fails. We have tests disabled elsewhere because they don't work yet. Why is this particular one not allowed for a feature hardly anyone uses and is also badly structured in the first place?